omero/sysadmins/unix/install-web/web-deployment.rst

```diff
@@ -102,20 +98,29 @@ configuration redirect the output of the following command into a file:
 .. literalinclude:: nginx-omero.conf
 
 
-OMERO.web requires ``body_in_file_only`` adjusted in your default nginx
```

Comment:
Hmmm.... since 4f5cc2a0baa89cfab2b99f38a0c9c565e6c81a25 in gh-1244 -- I don't have anything else either.